### PR TITLE
[css-flexbox] New aspect-ratio-intrinsic-size test

### PR DESCRIPTION
**Pull request by svillar**
This is a new test checking that an automatic physical width is always considered as definite, and thus, it should be used to compute the transferred size suggestion of flex items with aspect ratio. In this case the intrinsic sizes should be ignored for that computation.

### CODE REVIEW

**Comment by mrego** [comment]
This test is failing in all browsers right now, I don't know enough about flexbox to say if this is a bug in all of them, or if the test is wrong.

Maybe @dholbert, @davidsgrogan or @bfgeek can help to clarify this. Thanks.

File: css/css-flexbox/sspect-ratio-intrinsic-size-007-ref.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html>
```
Comment (mrego) [inline css/css-flexbox/sspect-ratio-intrinsic-size-007-ref.html:1]: Ok, this fails because this file has a wrong name **sspect** vs **aspect**.

**Comment by bfgeek** [comment]
Why do the two SVG have different aspect-ratios? (2 vs. 1.778)

**Comment by svillar** [comment]
> Why do the two SVG have different aspect-ratios? (2 vs. 1.778)

Fixed, good catch

**Review by davidsgrogan — APPROVED** [review]

**Pull request merged**